grid2op/tests/test_attached_envs.py

```diff
@@ -183,7 +183,7 @@ def test_action_space(self):
 
     def test_observation_space(self):
         assert issubclass(self.env.observation_space.subtype, CompleteObservation)
-        size_th = 467
+        size_th = 470
```

Comment:
why the size increase by 3 ? It has no alert and we don't add anything else, right ?